### PR TITLE
add documents for listing and querying item

### PR DESCRIPTION
**Pull request by wjduand**

### CODE REVIEW

File: service/service.py [hunk 1]
```diff
@@ -306,6 +311,45 @@ def wishlists_reset():
...
+    @api.expect(wishlist_item_args, validate=True)
+    @api.response(404, 'No wishlist item found.')
+    @api.marshal_list_with(wishlistProduct_model)
+    def get(self, wishlist_id):
+        """ Query a wishlist items from URL """
+        app.logger.info('Querying Wishlist items')
+        wishlist = Wishlist.find(wishlist_id)
+        if not wishlist:
+            api.abort(404, "Wishlist was not found.")
+        wishlist_id = request.args.get('wishlist_id')
```
Comment (nisargthakkar) [inline service/service.py]: I have 2 concerns here:
1) I think we have a bug in our code. Can we use `wishlist_id` from the request param? It doesn't make sense to call an api with a particular `wishlist_id` and allow some other `wishlist_id` to be set in the query params. We might need to fix some tests for this as well.
2) Another is that the `wishlist_id` hasn't been mentioned in the model. I think the previous tests were failing due to `required=True`. We could reintroduce that argument in `wishlist_item_args`, not change any tests right now, but change them in a later PR.

**Review by nisargthakkar — CHANGES_REQUESTED** [review]

File: service/service.py (hunk 1 above)
Comment (wjduand) [reply service/service.py]: I've tried to use the `wishlist_id` from the args, but when I execute the docs, it just keeps loading without any errors or logs.

Comment (nisargthakkar) [reply service/service.py]: Do you mean the args in `wishlist_item_args`? Or the query param args? Or the ones we pass in the api endpoint?

Comment (wjduand) [reply service/service.py]: In `wishlist_item_args`, when I defined it in the model, and use `wishlist_id = request.args.get('wishlist_id')` for all `wishlist_id`.

Comment (nisargthakkar) [reply service/service.py]: even with `required=False`?

Comment (wjduand) [reply service/service.py]: Yes

Comment (nisargthakkar) [reply service/service.py]: okay. That's interesting. So do you want to just go ahead and merge this and we do the fixing of the tests as a cleanup task?

Comment (wjduand) [reply service/service.py]: How do you want to change the tests? Like checking if the same wishlist_id is being used?

Comment (nisargthakkar) [reply service/service.py]: I was thinking not considering wishlist id from the query params (`request.args.get`). Just the url param should suffice.

Comment (wjduand) [reply service/service.py]: Ok I just changed the `wishlist_id` in service.py. I'm not sure if there are tests to be changed, but I will take a look tomorrow.

Comment (nisargthakkar) [reply service/service.py]: Thanks. All Tests pass. There shouldn't be anything necessary to change then

**Review by nisargthakkar — APPROVED** [review]
LGTM. We can increase the coveage later.

**Pull request merged**